docs/app/pages/components/components-sections/search/logical-expression-builder/logical-expression-builder.component.html

```diff
@@ -0,0 +1,331 @@
+                <code>label: string</code> - The text that is displayed in the UI.
+            </li>
+            <li>
+                <code>icon?: string</code> - Optionally, the name of a <code>ux-icon</code> that is displayed for this
+                option.
+            </li>
+        </ul>
+
+        <p>Additionally, a function to validate the entered value can be provided.</p>
+
+        <ul>
+            <li>
+                <code>validateFunction: (value: string[]) => boolean</code> - If specified, this
+                function is used to validate the value entered by the user.
+            </li>
+        </ul>
+    </tr>
+
+    <tr uxd-api-property name="TextInputComponent">
+        <p>
+            Contains a simple text input, expecting a value of type <code>string</code>. The <code>data</code> can contain
+            the following properties:
+        </p>
+
+        <ul>
+            <li>
+                <code>validateFunction: (value: string) => boolean</code> - If specified, this
+                function is used to validate the value entered by the user.
+            </li>
+        </ul>
+    </tr>
+</uxd-api-properties>
+
+<h4>Custom Input Components</h4>
+
+<p>
+    In addition to the predefined Input Components, it is also possible to built custom input components. These must
+    conform with the following interfaces:
+</p>
+
+<uxd-api-properties tableTitle="Outputs">
+    <tr uxd-api-property name="value" type="any">
+        Contains the value of the respective condition.
+    </tr>
+
+    <tr uxd-api-property name="data" type="any">
+        Contains the data for the specific <code>field</code> defined in the fields definition. E.g., it can be used
+        to pass options for a select input, or any additional data the input component needs that depends on the
+        specific field.
+    </tr>
+</uxd-api-properties>
+
+<uxd-api-properties tableTitle="Outputs">
+    <tr uxd-api-property name="valueChange" type="any">
+        This event will fire whenever the value has been modified. It will contain the latest value.
+    </tr>
+    <tr uxd-api-property name="valid" type="boolean">
+        This event will fire whenever the validity of the value has changed. It will contain <code>true</code> if the
+        value is valid, and <code>false</code> if it is not.
+    </tr>
+</uxd-api-properties>
+
+<h4>Example for a custom Input Component</h4>
+
+<p>
+    Below is an example for a custom Input Component containing a <code>textarea</code> input. In this specific case,
+    the <code>data</code> object is used to pass a <code>maxHeight</code> for the input.
+</p>
+
+<ux-tabset [minimal]="false">
+    <ux-tab heading="HTML">
+        <uxd-snippet [content]="snippets.compiled.textareaInputComponentHtml"></uxd-snippet>
+    </ux-tab>
+    <ux-tab heading="TypeScript">
+        <uxd-snippet [content]="snippets.compiled.textareaInputComponentTs"></uxd-snippet>
+    </ux-tab>
+</ux-tabset>
+
+<h4>Example Code</h4>
+
```

Comment:
Since there are two examples adjacent to one another, this should have a description clarifying that it is the code for the example at the top of the section.